### PR TITLE
Refactored code for add and remove functions

### PR DESCRIPTION
**Pull request by sawmon71293**
##  Update to-do-list
- add function and remove functions added

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,34 +1,77 @@
...
+
+const removeTask = (value) => {
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
+  const filtered = tasks.filter((task) => task.index !== parseInt(value, 10));
+  filtered.forEach((task, i) => {
+    task.index = i + 1;
+  });
+  localStorage.setItem('tasks', JSON.stringify(filtered));
+  loadTask(filtered);
+};
```
Comment (wayungi) [inline src/index.js]: - [ ] Your project requirements state that `Create a new JavaScript file for the new functionality`. Please define the functions for add, deleting & editing in this new file 👍🏾

Comment (wayungi) [reply src/index.js]: - [ ] There is no functionality for editing in your project. Kindly add one 🦖

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @sawmon71293 , 
image

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body 

## Required Changes ♻️
_Check the comments under the review._
- [ ] The last project requirement state that `All changes to the To Do List should be saved in local storage.` When I mark an item as complete on your app, the complete field still remains false.Please work on this 🥲 
false

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @sawmon71293, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**